packages/office-addin-dev-certs/src/verify.ts

```diff
@@ -31,13 +32,12 @@ export function isCaCertificateInstalled(): boolean {
 
   try {
     const output = execSync(command, { stdio: "pipe" }).toString();
-    if (process.platform === "darwin") {
-      return true;
-    } else if (output.length !== 0) {
-      return true; // powershell command return empty string if the certificate not-found/expired
+    // script files return empty string if the certificate not found or expired
+    if (output.length !== 0) {
+      return true; 
     }
   } catch (error) {
-    // Mac security command throws error if the certifcate is not-found/expired
+    // Some commands throw errora if the certifcate is not found or expired
```

Comment:
NIT: throws*

Reply:
Actually, "commands throw" is correct . . . but the "errora" should be "errors"